src/App.vue

```diff
@@ -1,14 +1,33 @@
 <template>
 	<div id="app">
-		<QueryBuilder/>
+		<QueryBuilder v-if="isi18nLoaded"/>
```

Comment:
if this is false, does that mean nothing is loaded ? 
as micheal suggested, we need some sort of Fallback mechanism.

Reply:
Yes but the i18n should be loaded for that language first, we can implement falling back chain of languages later (you load that language and English and if the message doesn't exist, vue-banana automatically falls back to English)